test/extended/baremetal/hosts.go

```diff
@@ -186,4 +186,78 @@ var _ = g.Describe("[sig-installer][Feature:baremetal][Serial] Baremetal platfor
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(found).To(o.BeFalse())
 	})
+
+	g.It("configure BIOS settings during cleaning", func() {
+		var procTurboMode = "ProcTurboMode"
+
+		skipIfNotBaremetal(oc)
+
+		dc := oc.AdminDynamicClient()
+		hostData, secretData := helper.GetExtraWorkerData(0)
+
+		// Deploy extra worker and wait
+		host, _ := helper.CreateExtraWorker(hostData, secretData)
+		host = helper.WaitForProvisioningState(host, "available")
+		hostName := getStringField(*host, "baremetalhost", "metadata.name")
+
+		hfsClient := hostfirmwaresettingsClient(dc)
+		hfs := getHostFirmwareSettings(hfsClient, hostName)
+
+		status, _, err := unstructured.NestedStringMap(hfs.Object, "status", "settings")
+		o.Expect(err).NotTo(o.HaveOccurred())
+		spec, _, err := unstructured.NestedStringMap(hfs.Object, "spec", "settings")
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		// Change HostFirmwareSetting to an invalid value
+		spec[procTurboMode] = "Foo"
+		g.By(fmt.Sprintf("setting firmwaresetting %s invalid value for host %s", procTurboMode, hostName))
+		err = unstructured.SetNestedStringMap(hfs.Object, spec, "spec", "settings")
+		o.Expect(err).NotTo(o.HaveOccurred())
+		_, err = hfsClient.Update(context.Background(), hfs, metav1.UpdateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		time.Sleep(2 * time.Second)
+		hfs = getHostFirmwareSettings(hfsClient, hostName)
+		checkConditionStatus(*hfs, "Valid", "False")
```

Comment:
It would be better to add `g.By` to describe the purpose of this check?

Reply:
Will add.